### DISCUSSION TITLE
Geomean Matlab Change

### DISCUSSION THREAD

**Discussion by PenguinPeeps** [DrCoffey/DeepSqueak#215] [Q&A]
Hello!

Recently an error keeps popping up in MatLab for the geomean function when using DeepSqueak. It reads:

"> In CalculateStats (line 11)
In update_focus_display (line 25)
In update_fig (line 48)
In callBoxDeleteCallback>choosedialog/label_callback (line 90)
In uiwait (line 81)
In callBoxDeleteCallback>choosedialog (line 81)
In callBoxDeleteCallback (line 22)
In images.roi.internal/ROI/determineSelectionStatus
In images.roi.internal/ROI/startROIDrag
In images.roi.internal.ROI 
Warning: The function "geomean" has been renamed "geo_mean". Please update MATLAB to before it is discontinued"

I considered checking that line of code myself, but I'm pretty green in that department and my project is precious.

Thanks!

**Comment by DrCoffey** [comment]
Thanks for spotting this, it is just a warning correct? I'll try to fix it before geomean is discontinued.

**Reply by PenguinPeeps** [reply]
Correct, it is just a warning. Thank you!
